docs/competition/matches/match-scoring.md

```diff
@@ -11,47 +11,76 @@ Any roles related to scoring matches go here.
+- Ensuring no team members touch a robot after it has first moved. (see 'Teams touching robots').
+- Switching off robots when a team requests that you do so, or if they do not automatically turn off after a match ends.
+- Resetting the arena by moving props (tokens, for example) back to the correct place after you and your partmer have both recorded the end-state of the match. There is no mechanism for delaying the matches, so please request extra help if you are struggling to make the deadlines.
 
-Teams must not be allowed in the arena during a match and only one team member is allowed in the area between the crowd barrier and arena wall. It is the responsibility of the team members to place their robot in the start zone before the match starts. If a robot moves off before the match starts (aside from minor servo-resetting arm movements) teams are allowed to switch off their robot, reset it and start again. However, if the mis-started robot touches any arena prop then they are immediately disqualified. Once the robot start button has been pressed and the match has started, teams are not allowed to interact with their robots until after scoring has taken place.
+You are not responsible for:
+- Making sure a match starts with the correct number of teams -- that is down to the shepherds.
 
-If the team members indicates that they wish for you to turn off their robot (i.e. "throwing in the towel") you may do so but by doing so they forfeit the match which will be recorded in the score sheets as a disqualification. There are two cases where you are allowed to switch off a robot yourself, one is when the match ends and the other is when a robot is causing **unintentional and significant** damage to another robot (i.e. mashes into another robot, pins it, stopping it from being able to function -- includes one dragging the other around if they become intertwined), and in that case it is up to you to decide which robot should be switched off but their match is not forfeited. If the damage is intentional, then the team will get forfeited (potentially for previous matched too), but this decision should be left to the judge. A checklist for robot damage is available below.
+### Summary of the most important rules & regulations for SR2019:
 
-You must read [the rules](https://www.studentrobotics.org/docs/rules/) before starting this role to make sure you understand the game, the scoring rules and how the arena is laid out.
+- A token is considered to be ‘in’ a zone if either:
+  - Three corners of the token are in contact with the floor area inside the zone. Corners touching tape are not in the zone, tokens on Robots, or standing on their edges are not in the zone.
+  - The token is touching *only* other tokens (not robots, not the floor), which themselves are 'in'.
+
+- A robot is considered to be 'in' a zone if:
+  - The exact center of the flagpole is above the zone (if it's directly above the coloured tape, it's not in the zone).
```

Comment:
This wasn't how I'd read the rules. I'd read the rule as being "look along the cylinder which is the flagpole and see what zone you can see".

However on reading up on what a "centroid" is, I'm not sure I've understood correctly, however I'm not sure what the rule means if it doesn't mean what I'd interpreted it as.

@prophile could you comment on whether this is correct?

Reply:
I think this is the discrepancy HRS was talking about in the forum

Reply:
(Otherwise you could use the flag as a laser pointer to tactically aim at the chosen finish location)

Reply:
Got confirmation from Alistair that my wording is correct